src/FetchLoader.js

```diff
@@ -38,39 +28,34 @@ class FetchLoader extends React.Component {
     )
   }
 
-  _fetchAndSetState(host, resource) {
+  async componentDidMount() {
```

Comment:
Nice! I didn't know you could make the `componentDidMount` async! I guess it makes sense.

Reply:
Yes, I thought that in principle `async` just lets you use `await` to block a promise, and there’s nothing else about it. I find it a bit weird that it’s not a pattern mentioned in React’s official documentation.